test/regression/external/notify-callback.spec.js

```diff
@@ -0,0 +1,25 @@
+const { getLastNotifications, simulateNotifyCallback } = require('../shared/helpers/notifications');
+const { sleepFor } = require('../shared/helpers/utils');
+const config = require('./config');
+
+/* eslint-disable no-undef */
+describe('notify callback attempt', function () {
+  let scheduledNotification, scheduledNotificationAfterCallback, scheduledNotificationNotifyId;
+  before(async () => {
+    await sleepFor(1000);
+    scheduledNotification = await getLastNotifications(config.baseUrl, config.userEmails.notifyCallbackTestEmail);
+    console.log(scheduledNotification);
+    scheduledNotificationNotifyId = JSON.parse(scheduledNotification).data[0].notify_id;
+  });
+
+  it('checks if the notification was found', () => {
+    expect(scheduledNotificationNotifyId.length).toBe(36);
+  });
+
+  it('calls the notify callback endpoint', async () => {
+    await simulateNotifyCallback(scheduledNotificationNotifyId);
+    await sleepFor(1000);
+    scheduledNotificationAfterCallback = await getLastNotifications(config.baseUrl, config.userEmails.notifyCallbackTestEmail);
+    expect(JSON.parse(scheduledNotificationAfterCallback).data[0].notify_status).toBe('delivered');
```

Comment:
Would it be cleaner to abstract away parsing the JSON data and returning `data` from the `getLastNotifications` helper function?

Reply:
Yep. That makes sense. I will do that.